### PR TITLE
Organize and rename stories

### PR DESCRIPTION
**Pull request by pnavarrc**
Storybook shows stories sorted by how they are imported, not by name. I updated the config to sort them alphabetically. I also grouped the stories in "Components" and "Location Page" for now, we had a few groups with names that weren't super intuitive.

Most of the changes in this PR are changes in the title of stories.

## Changes
- Update the Storybook config to sort stories alphabetically
- Organize the stories into "Components" and "Location Page"
- Updated some stories to use the [Component Story Format](https://storybook.js.org/docs/react/api/csf) instead of mdx 
- Removed a few unused buttons (I checked that they are not used) 
- Remove the deprecated DLL flags on the storybook script entry in `package.json`

### CODE REVIEW

File: src/components/Button/Button.style.ts [hunk 1]
```diff
@@ -1,56 +1,5 @@
...
-
-const disableRippleProps = {
-  disableRipple: true,
-  disableElevation: true,
-  disableFocusRipple: true,
-};
-
-/**
- * Button that looks like a link.
- */
```
Comment (pnavarrc) [inline src/components/Button/Button.style.ts:13]: These buttons are not used anywhere - The BaseButton here should probably be removed in favor of the design system buttons.

File: src/components/ExpandableContainer/ExpandableContainer.stories.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { useCcviForFips } from 'common/hooks';
 import { EventCategory } from 'components/Analytics';
 
 export default {
-  title: 'Below the fold/Expandable container',
+  title: 'Location Page/Expandable container',
```
Comment (chasulin) [inline src/components/ExpandableContainer/ExpandableContainer.stories.tsx:9]: expandable container could be part of `Components`

File: src/components/VulnerabilitiesBlock/CcviThermometer/CcviThermometer.stories.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import CcviThermometer from './CcviThermometer';
 
 export default {
-  title: 'Shared Components/Vulnerabilities/CcviThermometer',
+  title: 'Components/Vulnerabilities/CcviThermometer',
```
Comment (chasulin) [inline src/components/VulnerabilitiesBlock/CcviThermometer/CcviThermometer.stories.tsx:5]: `Location Page` would make more sense for this one

File: src/components/VulnerabilitiesBlock/ThemesBlock/ThemesBlock.stories.tsx [hunk 4]
```diff
@@ -3,7 +3,7 @@ import ThemesBlock from './ThemesBlock';
 import { useCcviForFips } from 'common/hooks';
 
 export default {
-  title: 'Shared Components/Vulnerabilities/ThemesBlock',
+  title: 'Components/Vulnerabilities/ThemesBlock',
```
Comment (chasulin) [inline src/components/VulnerabilitiesBlock/ThemesBlock/ThemesBlock.stories.tsx:6]: `Location Page` would make more sense

File: src/components/VulnerabilitiesBlock/VulnerabilitiesBlock.stories.tsx [hunk 5]
```diff
@@ -4,7 +4,7 @@ import { useCcviForFips } from 'common/hooks';
 import regions from 'common/regions';
 
 export default {
-  title: 'Shared Components/Vulnerabilities/VulnerabilitiesBlock',
+  title: 'Components/Vulnerabilities/VulnerabilitiesBlock',
```
Comment (chasulin) [inline src/components/VulnerabilitiesBlock/VulnerabilitiesBlock.stories.tsx:7]: `Location Page` would make more sense

File: src/components/VaccinationEligibilityBlock/VaccinationEligibilityBlock.stories.tsx [hunk 6]
```diff
@@ -3,7 +3,7 @@ import VaccinationEligibilityBlock from './VaccinationEligibilityBlock';
 import regions from 'common/regions';
 
 export default {
-  title: 'Shared Components/VaccinationEligibilityBlock',
+  title: 'Components/VaccinationEligibilityBlock',
```
Comment (chasulin) [inline src/components/VaccinationEligibilityBlock/VaccinationEligibilityBlock.stories.tsx:6]: `Location Page`

**Pull request merged**